contracts/test/unit/oracle/VoltSystemOracle.t.sol

```diff
@@ -0,0 +1,212 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {Vm} from "./../utils/Vm.sol";
+import {DSTest} from "./../utils/DSTest.sol";
+import {getCore, getAddresses, FeiTestAddresses} from "./../utils/Fixtures.sol";
+import {MockScalingPriceOracle} from "../../../mock/MockScalingPriceOracle.sol";
+import {VoltSystemOracle} from "../../../oracle/VoltSystemOracle.sol";
+import {ScalingPriceOracle} from "../../../oracle/ScalingPriceOracle.sol";
+import {MockChainlinkToken} from "../../../mock/MockChainlinkToken.sol";
+import {Decimal} from "./../../../external/Decimal.sol";
+import {Constants} from "./../../../Constants.sol";
+import {SafeCast} from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import {LinkTokenInterface} from "@chainlink/contracts/src/v0.8/interfaces/LinkTokenInterface.sol";
+
+contract VoltSystemOracleTest is DSTest {
+    using Decimal for Decimal.D256;
+    using SafeCast for *;
+
+    /// @notice reference to the volt system oracle
+    VoltSystemOracle private voltSystemOracle;
+
+    /// @notice increase price by 3.09% per month
+    int256 public constant monthlyChangeRateBasisPoints = 309;
+
+    /// @notice increase the volt target price by 2% annually
+    uint256 public constant annualChangeRateBasisPoints = 200;
+
+    /// @notice block time at which the VSO will be able to be initialized and start accruing interest
+    uint256 public constant startTime = 100_000;
+
+    /// @notice starting oracle price
+    uint256 public constant startPrice = 1.0387e18;
+
+    Vm public constant vm = Vm(HEVM_ADDRESS);
+    FeiTestAddresses public addresses = getAddresses();
```

Comment:
Think we'll need to merge this branch with develop, this has since been renamed to `VoltTestAddresses`

Reply:
will do